backend/app/services/email/email_service_provider.py

```diff
@@ -0,0 +1,8 @@
+from app.services.interfaces.email_service_provider import IEmailServiceProvider
```

Comment:
i think we should move the interfaces folder directly under the app directory. Let me know if yall agree.